onnxruntime/core/session/inference_session.h

```diff
@@ -510,6 +510,10 @@ class InferenceSession {
   const long long kDurationBetweenSending = 1000 * 1000 * 60 * 10;  // duration in (us).  send a report every 10 mins
   std::string event_name_;                                          // where the model is loaded from: ["model_loading_uri", "model_loading_proto", "model_loading_istream"]
 
+  TimePoint time_sent_last_evalutation_start_;
+  const long long kDurationBetweenSendingEvaluationStart = 1000 * 50; // duration in (us). send a EvaluationStop Event every 50 ms;
```

Comment:
>duration in (us). send a EvaluationStop Event every 50 ms; [](start = 73, length = 58)

how did you decide on this number? can you add a comment saying why it's every 50 ms?